Doc/tutorial/inputoutput.rst

```diff
@@ -300,11 +300,14 @@ writing. The *mode* argument is optional; ``'r'`` will be assumed if it's
 omitted.
 
 Normally, files are opened in :dfn:`text mode`, that means, you read and write
-strings from and to the file, which are encoded in a specific encoding. If
-encoding is not specified, the default is platform dependent (see
-:func:`open`). ``'b'`` appended to the mode opens the file in
-:dfn:`binary mode`: now the data is read and written in the form of bytes
-objects.  This mode should be used for all files that don't contain text.
+strings from and to the file, which are encoded in a specific *encoding*.
+If *encoding* is not specified, the default is platform dependent
+(see :func:`open`).
+But passing ``encoding="utf-8"`` is highly recommended because
+UTF-8 is the most commonly used encoding for now.
```

Comment:
general rule: don't start a sentence with "and, but, so, or then" words.  and try not to end with "for now".

```
Explicitly passing ``encoding='utf-8'`` is recommended if that is what you need as it is the most common text encoding in the world and leaves no room for doubt about your code's intent.
```

perhaps.

Reply:
This is tutorial and reader won't know what they need.
I want to teach that UTF-8 is the first choice.

How about this?

> ``encoding="utf-8"`` is recommended unless you need to use other encoding because UTF-8 is the de-facto standard nowadays.

Reply:
How about:
`Because UTF-8 is the modern de-facto standard, ``encoding="utf-8"`` is recommended unless you know that you need to use a different encoding.`